argo-cd-apps/base/member/infra-deployments/kyverno/kyverno.yaml

```diff
@@ -30,14 +30,28 @@ spec:
             values: |
```

Comment:
We should consider adding [`namespaceOverride`](https://github.com/kyverno/kyverno/blob/release-1.13/charts/kyverno/values.yaml#L55-L56) to our values, since we override the namespace we're deploying to.

Reply:
will do in #5414